package.json

```diff
@@ -122,7 +123,6 @@
 		"typescript": "^2.9.2",
 		"uglify-js": "^3.4.4"
 	},
-	"dependencies": {},
```

Comment:
Best case scenario this does nothing. Worst case, this breaks a bunch of stuff. I'd prefer you didn't remove this.